### PR TITLE
Add weather emojis

### PR DESCRIPTION
**Pull request by williamvwang**
Added preliminary logic for switching skycodes for weather-related emojis

### CODE REVIEW

File: public/javascript/getWeather.js [hunk 1]
```diff
@@ -19,20 +19,68 @@ module.exports = function getWeather(api, threadID, body) {
...
+						if (index !== 0) {
+							message += ' Precipitation: ' + day.precip + '%';
+						}
 					});
 					api.sendMessage(message, threadID);
 					console.log('Weather info sent to ' + threadID);
 				}
 			});
-			if (end != undefined) {
+			if (end != null) {
```
Comment (renxinhe) [inline public/javascript/getWeather.js:83]: I thought the `end` function is undefined if `sendTypingIndicator` gives an error.

File: public/javascript/db.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ var mongoose = require('mongoose');
 if (process.env.MONGODB_URI == undefined) {
 	return console.error('No database found.')
 } else {
-	mongoose.connect(process.env.MONGODB_URI, function(err){
+    var url = process.env.MONGODB_URI;
```
Comment (renxinhe) [inline public/javascript/db.js:11]: Fix indentation.

File: public/javascript/db.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * @file dj.js
```
Comment (renxinhe) [inline public/javascript/db.js:2]: DDDDJJJJJ KKHAAAAALEDDD! 🔑🔑🔑🔑🔑

File: public/javascript/getWeather.js (hunk 1 above)
Comment (williamvwang) [inline public/javascript/getWeather.js:83]: [== null](https://dorey.github.io/JavaScript-Equality-Table/) is the standard way to catch `null` and `undefined` simultaneously

Comment (renxinhe) [inline public/javascript/getWeather.js:83]: 🔑 web dev 🔑

**Pull request merged**